src/features/meetings/monthly_view/index.tsx

```
@@ -0,0 +1,1157 @@
+            <Typography
+              className="h4"
+              color={'var(--text-treasures-from-gods-word)'}
+              width={`${196}px`}
+            >
+              {t('tr_tgwTalk')}
+            </Typography>
+            {selectedWeeks.map((value, index) => {
+              return (
+                <>
+                  <WeekHoverBox type={'tgw_talk'} week={value}>
+                    <PersonSelector
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)